interlok-jclouds-blobstore/src/main/java/com/adaptris/jclouds/blobstore/BlobStoreConnection.java

```diff
@@ -156,16 +106,10 @@ protected void stopConnection() {
   }
 
   @Override
-  @SuppressWarnings("deprecation")
   protected void closeConnection() {
-    IOUtils.closeQuietly(context);
+    IOUtils.closeQuietly(context, null);
```

Comment:
It always bugged me that IOUtils didn't have a variation that took `AutoCloseable` so there is Closer.closeQuietly() available. It makes no difference mind